src/app-layout/__integ__/app-layout-focus-delegation.test.ts

```diff
@@ -6,17 +6,32 @@ import useBrowser from '@cloudscape-design/browser-test-tools/use-browser';
 import createWrapper from '../../../lib/components/test-utils/selectors';
 import { viewports } from './constants';
 
+class FocusDelegationPage extends BasePageObject {
+  getActiveElement() {
+    return this.browser.execute(() => {
+      return document.activeElement;
+    });
+  }
+}
+const testIf = (condition: boolean) => (condition ? test : test.skip);
+
 const wrapper = createWrapper().findAppLayout();
 
 function setupTest(
-  testFn: (page: BasePageObject) => Promise<void>,
-  { pageName = 'with-split-panel', visualRefresh = false, splitPanelPosition = '', mobile = false }
+  testFn: (page: FocusDelegationPage) => Promise<void>,
+  {
+    pageName = 'with-split-panel',
+    visualRefresh = false,
+    splitPanelPosition = '',
+    mobile = false,
+    appLayoutWidget = false,
+  }
 ) {
   return useBrowser(async browser => {
     const url = `#/light/app-layout/${pageName}?visualRefresh=${visualRefresh}${
       splitPanelPosition ? `&splitPanelPosition=${splitPanelPosition}` : ''
-    }`;
-    const page = new BasePageObject(browser);
+    }${appLayoutWidget ? `&appLayoutWidget=${appLayoutWidget}` : ''}`;
```

Comment:
This URL concatenation logic becomes too complex, time to use `URLSearchParams` helper: https://github.com/cloudscape-design/components/blob/1b116cef645c93b692a4aaf63cb36360d954a32a/src/app-layout/__integ__/app-layout-split-panel.test.ts#L99-L103

Reply:
Like this way better. Thanks for suggesting